apps/web/src/app/(dashboard)/documents/data-table-action-dropdown.tsx

```diff
@@ -109,6 +118,10 @@ export const DataTableActionDropdown = ({ row }: DataTableActionDropdownProps) =
     window.URL.revokeObjectURL(link.href);
   };
 
+  const handleDelete = async () => {
```

Comment:
Can you rename `handleDelete` to `onDraftDelete` or something instead? 

We always prefer to use the `on<ActionName>` naming convention for event handlers.